docs/general-usage/model-types.rst

```diff
@@ -45,6 +45,67 @@ GraphQLString
         }
 
 
+GraphQLCollection
+-------------
+.. module:: grapple.models
+.. class:: GraphQLCollection(nested_type, *args, **kwargs)
+
+    A field type that wraps another model type in a list, Most likely to be used when referencing ForeignKey lists (like when using Orderables).
```

Comment:
"A field type that wraps another model type in a list. Best suited for referencing `Orderable`s (i.e. `ForeignKey` lists)."

Reply:
Done.